src/main/java/com/teragrep/lsh_01/pool/Pool.java

```diff
@@ -109,4 +109,14 @@ public void close() {
         offer(stub);
     }
 
+    @Override
```

Comment:
i wonder if this makes sense here at all. pools must always be unique with their queue i.e. no two pools must share the same and rest is just the encapsulated details about the said queue.

Reply:
Agreed, I took it out.